packages/storybook-webpack-angular-types-plugin/src/lib/webpack-angular-types-plugin/plugin.ts

```diff
@@ -172,6 +144,48 @@ export class WebpackAngularTypesPlugin {
 		return groupedExportsInformation;
 	}
 
+	private addToGroupedExportsInformation(
+		exportsInformationMap: Map<string, ExportsInformation>,
+		{
+			functionInformation,
+			constantInformation,
+		}:
+			| { functionInformation: FunctionInformation; constantInformation?: never }
+			| { constantInformation: ConstantInformation; functionInformation?: never },
+	): Map<string, ExportsInformation> {
```

Comment:
Why does the type need to be so complex? Can't you use the base interface of those two types? You only need the name and the groupBy information.

Reply:
It's added under the "functions" or "constants" category in the table based on this separation. See line 228. But we can simplify it and remove the categories if you'd like that better.

Reply:
I refactored it again in the latest commit to get rid of the strange type. Also, I moved it to an own file "group-export-information.ts"

Reply:
Looks better now 👍 
Also keeps the entry point plugin file a bit more concise 👍